contributors/guide/owners.md

```diff
@@ -86,6 +86,28 @@ Instead, set a `.*` key inside `filters` (as shown in the previous example).
 **WARNING**: The `approve` plugin [does not currently respect `filters`][test-infra-7690].
 Until that is fixed, `filters` should only be used for the `labels` key (as shown in the above example).
 
+
+#### Emeritus
+
+It is inevitable, but there are times when someone may shift focuses, change jobs or step away from
+a specific area in the project for a time. These people may be domain experts over certain areas
+of the codebase, but can no longer dedicate the time needed to handle the responsibilities of
+reviewing and approving changes. They are encouraged to add themselves as an _"emeritus"_ approver
+under the `emeritus_approvers` key.
+
+Any GitHub username under the `emeritus_approvers` key will be ignored by prow for assignment.
```

Comment:
One might interpret this to mean that prow will not use `emeritus_approvers` to assign people work while still granting `emeritus_approvers` the ability to approve code.

If so then this seems even better than being a regular approver -- all the same power without the hassle of helping others.

Reply:
Can call that out more explicitly. Something like:
"GitHub usernames listed under the `emeritus_approvers` key can no longer approve code (use the `/approve` command) and will be ignored by prow for assignment."